hedy.py

```diff
@@ -186,6 +186,22 @@ def _translate_index_error(code, list_name):
         """)
 
 
+def translate_value_error(command, value, suggestion_type):
```

Comment:
We might want to call this translate_runtime_error? I think that is a bit more clear